addons/website/static/src/snippets/s_table_of_content/000.js

```diff
@@ -18,6 +19,22 @@ const TableOfContent = publicWidget.Widget.extend({
         this._updateTableOfContentNavbarPosition();
         this._updateTableOfContentNavbarPositionBound = this._updateTableOfContentNavbarPosition.bind(this);
         extraMenuUpdateCallbacks.push(this._updateTableOfContentNavbarPositionBound);
+
+        this.wasVisible = this.$target.is(':visible');
+        this.resizeObserver = new ResizeObserver(entries => {
+            entries.forEach(entry => {
+                if (entry.contentRect.width === 0 && this.wasVisible) {
+                    wUtils.disposeScrollSpy(this);
+                    wUtils.activateScrollSpy(this,
+                        (tocEl) => tocEl === this.$target[0].closest('.s_table_of_content'));
+                    this.wasVisible = false;
+                } else {
+                    this.wasVisible = true;
+                    this._updateTableOfContentNavbarPosition();
+                }
+            });
+        });
+        this.resizeObserver.observe(this.$target[0]);
```

Comment:
In this end, I am not sure about the this need of having to think about table of content everywhere some visibility value changes... :grimacing: Same for the need of a resize observer for the table of content, seems overkill. You could probably just listen to the window resize via the utils of config.js / something like that.

But mainly: is there no way to properly patch the lib (the jquery change you explain in your commit message) to avoid all of this? Maybe even check if they did not solve the issue in a further update of bootstrap, in which case we could really patch the current lib in place?

Reply:
Indeed it is way more simple and robust to go with a lib patch :+1: